### ISSUE TITLE
Jenkins: build slaves are running out of disk space.

### ISSUE THREAD

**Issue by brucellino** [AAROC/CODE-RADE#4]
We've seen many  builds failing with disk full errors.

I've created the NFS share on the CI server (Jenkins itself), to export the `/repo`directory which builds are compiled _into_. However the disk is still filling up. 

Investigating.

**Comment by brucellino** [comment]
On both slaves, the `/var` directory grows to > 30 GB. This is probably due to old builds. 

On SL6, e.g. the size of the workspaces are :

```
343M    atlas-deploy
2.9G    boost-deploy
233M    cmake-deploy
21M fftw3-deploy
4.5G    gcc-4.8.2
16G gcc-deploy
48M gmp-deploy
191M    gsl-1.16
1.9G    heasoft-deploy
168K    lapack-deploy
11M mpc-1.0.1
11M mpc-deploy
26M mpfr-deploy
35M ncurses-deploy
22M numpy-deploy
3.2G    OpenMPI
410M    python-deploy
```

We need to clean out the workspaces and ensure that fewer builds are kept.

**Comment by brucellino** [comment]
On further thought, I think a better way would be to export the workspace from the CI server, to the nodes, since there is plenty of space on the CI server. I'm busy moving the content from the slaves to the CI server and updating the NFS exports.

**Comment by brucellino** [comment]
I have fixed the problem, I think.
1. Moved everything on the nodes under `/var/lib/jenkins/workspace` on the nodes to the same directory on the ci server.
2. removed the contents on the nodes.
3. Changed ownership of that directory on the ci server to `apprepo` - to keep gid and uid the same on the slaves.
4. added an export on the ci to export this directory to the two machines.
5. added an entry in fstab to mount this directory on the slaves.

then, I restarted the services (nfs) and re-mounted the directory. Both slaves have around 30 GB free. Re-starting the build process.
